content/sensu-go/6.7/observability-pipeline/observe-entities/entities.md

```diff
@@ -356,6 +356,156 @@ spec:
+    libc_type: glibc
+    network:
+      interfaces:
+        - addresses:
+            - 127.0.0.1/8
+          name: lo
+        - addresses: null
+          name: tunl0
+        - addresses: null
+          name: ip6tnl0
+        - addresses:
+            - 172.18.0.4/16
+          mac: 02:42:ac:12:00:04
+          name: eth0
+    os: linux
+    platform: redhat
+    platform_family: rhel
+    platform_version: '7.9'
+    processes: null
+    vm_role: guest
+    vm_system: ''
+{{< /code >}}
+
+{{< code json >}}
+{
+  "type": "Entity",
+  "api_version": "core/v2",
+  "metadata": {
+    "name": "6b6264feda40",
+    "namespace": "sensu-system"
+  },
+  "spec": {
+    "deregister": false,
+    "deregistration": {
+    },
+    "entity_class": "backend",
+    "last_seen": 0,
+    "sensu_agent_version": "",
+    "subscriptions": null,
+    "system": {
+      "arch": "amd64",
+      "cloud_provider": "",
+      "hostname": "6b6264feda40",
+      "libc_type": "glibc",
+      "network": {
+        "interfaces": [
+          {
+            "addresses": [
+              "127.0.0.1/8"
+            ],
+            "name": "lo"
+          },
+          {
+            "addresses": null,
+            "name": "tunl0"
+          },
+          {
+            "addresses": null,
+            "name": "ip6tnl0"
+          },
+          {
+            "addresses": [
+              "172.18.0.4/16"
+            ],
+            "mac": "02:42:ac:12:00:04",
+            "name": "eth0"
+          }
+        ]
+      },
+      "os": "linux",
+      "platform": "redhat",
+      "platform_family": "rhel",
+      "platform_version": "7.9",
+      "processes": null,
+      "vm_role": "guest",
+      "vm_system": ""
+    }
+  }
+}
+{{< /code >}}
+
+{{< /language-toggle >}}
+
+### Access backend entities
+
+Only cluster admins have access to the `sensu-system` namespace and the backend entities it contains.
+
+If you have cluster admin permissions, you can use [sensuctl][42] and the [web UI][43] to access backend entities like other entities.
+
+Cluster admins who have write permissions for the `sensu-system` namespace can edit **only** labels and subscriptions for backend entities.
+
+### Backend entities and agent information
+
+Sensu uses the same algorithm to determine backend entity names as the agent uses to determine entity names.
+
+If a backend and an agent try to create the same entity, the entity class will default to `backend`.
+The information provided by the backend takes precedence over the information provided by the agent.
+The backend should update backend entities to use information from the backend instead of from the agent.
+
+### Backend entity keepalives
+
+Backend entities generate a keepalive event every ...
+
+**TODO**: Need more information about keepalives from backend entities. Also, this question from the original issue: "Question: What should the behaviour be when there are both a backend and an agent generating keep-alive events?"
+
```

Comment:
I couldn't find answers to the questions in https://github.com/sensu/sensu-go/issues/4443#issuecomment-1020384973 so I added placeholders here and hopefully you can answer in review